ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -113,6 +163,62 @@ const triggerSet: OopsyTriggerSet<Data> = {
           delete data.raiseTracker[matches.targetId];
       },
     },
+    {
+      id: 'General Overwritten Boss Mit',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: targetMitAbilityIds.concat(partyMitAbilityIds) }),
+      mistake: (data, matches) => {
+        const isTargetMit = targetMitAbilityIds.includes(matches.id);
+        const isPartyMit = partyMitAbilityIds.includes(matches.id);
+        if (isTargetMit && matches.targetId === 'E0000000') // missed
+          return;
+        if (isPartyMit && !data.party.partyIds_.includes(matches.sourceId))
+          return;
+        if (isPartyMit && matches.targetId !== matches.sourceId)
+          return;
+        data.mitTracker ??= {};
+        const target = isTargetMit ? matches.targetId : 'party';
+        const lastTime = data.mitTracker[target]?.[matches.id]?.time;
+        const lastSource = data.mitTracker[target]?.[matches.id]?.source;
```

Comment:
Sorry for changing my mind here, but in looking at this, I think I'd prefer not to have a special "party" value here. It seems easy to get wrong (or misspell). Can you split the party mit out into a separate object so it can be more properly type checked?

e.g. something like this

```typescript
type MitTracker = { [abilityId: string]: {
    time: number;
    source: string;
  };
};

export interface Data extends OopsyData {
  // etc
  targetedMitTracker?: {
    [targetId: string]: MitTracker;
  };
  partyMitTracker?: MitTracker;
```



and then you could do something more like this

```suggestion
        const mitTracker = isTargetMit ? (targetedMitTracker[target] ??= {}) : partyMitTracker;
        const lastTime = mitTracker[matches.id]?.time;
        const lastSource = mitTracker[matches.id]?.source;
```